src/I18n/French.elm

```diff
@@ -0,0 +1,110 @@
+module I18n.French exposing (..)
```

Comment:
I didn't recall we had the generated files in the repo, I believe this has to do with this being a library :thinking: